### DISCUSSION TITLE
Azure Monitor Metrics sink?

### DISCUSSION THREAD

**Discussion by ryanhendersonabs** [vectordotdev/vector#17245] [Q&A]
Does vector have a sink that could send metrics to Azure Monitor Metrics? I see that there's an Azure Monitor Logs sink, but nothing for metrics. https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/data-platform-metrics

**Comment by bruceg** [comment]
No, but we have an issue tracking this as a feature request. #10231
